aten/src/ATen/native/cpu/Intrinsics.h

```diff
@@ -22,6 +22,9 @@
     (defined(__VEC__) || defined(__ALTIVEC__))
 /* XLC or GCC-compatible compiler, targeting PowerPC with VMX/VSX */
 #include <altivec.h>
+#undef bool
```

Comment:
Same thing, please add a comment here, so people know why we undef those. Thanks.